openstack/compute/v2/extensions/hypervisors/results.go

```diff
@@ -62,6 +62,27 @@ func (r *Service) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// Server represents an instance running on the hypervisor
+type Server struct {
+	Name string `json:"name"`
+	UUID string `json:"uuid"`
+}
+
+func (r *Server) UnmarshalJSON(b []byte) error {
```

Comment:
This method isn't needed since it's not doing any custom logic within the body. You should be able to remove this whole method and not see any errors.